function Contribution() {
  return (
    <div>
      <div class="titleScreen1">
        <h1 class="titlescreentext">CONTRIBUTION</h1>
      </div>
      <div class="sizedbox60" />
      <div class="body">
        <h1 class="header1">Contribution</h1>
        <div class="sizedbox20"></div>
        <p class="indented">
          The final results from both biopanning and ELISA that we received from
          Kasetsart University are arranged to be provided for future iGEM
          teams. In our test to find the most specific nanobody that binds with
          the solanacearum, we were able to identify it out of the 20 PBS
          clones. We were able to demonstrate more effective use of both TB1 and
          HB2151. Future teams could implement this technique to make biopanning
          even more effective. Another advantage of our technique was the use of
          Tween-20 which acts as a detergent to decrease background staining.
          The phosphate-buffered saline solution that we used and calculated
          could be used for future teams to get the highest yield of specific
          nanobodies. Finally, our peptide can also be utilized by future teams.
        </p>

        <div class="sizedbox40" />
        <h1 class="header1">Characterizing existing parts BBa_K1223006</h1>
        <div class="sizedbox20"></div>
        <p class="indented">
          We contributed to future iGEM teams by completing a characterization
          on the existing part
          <a href="http://parts.igem.org/Part:BBa_K1223006" class="blue">
             
            <u>BBa_K1223006</u>
          </a>
          .
        </p>
        <div class="sizedbox20" />
        <p class="indented">
          Characterization of 
          <a href="http://parts.igem.org/Part:BBa_K1223006" class="blue">
             
            <u>BBa_K1223006</u>
          </a> 
          TRIS_iGEM team
        </p>
        <div class="sizedbox20" />
        <p>
          <b>Background:</b>
          <br />
        </p>
        <p class="indented">
          Similarly to what the 2019 Madrid team proposed, this part holds
          potential to be used in other ways of separation. For our experiment,
          the Histidine tag was used as a part of purification through His
          column. Since Histidine contains polar properties, metal ions such as
          Ni2+ or Cu2+, will attract the histidine and allow for bonds to form.
          This attraction will form strong bonds that will cause the His-tags to
          form bonds with the linings on the side of the column, allowing for
          washing of surplus proteins, without losing our target protein.
        </p>
        <div class="sizedbox20" />
        <p>
           
          <b>Experiment:</b> 
        </p>
        <p class="indented">
          For our experiment, we had to undergo sonication to remove our protein
          from the host cell itself. However, when sonication occurs, even
          though the host cell is destroyed and proteins are free, all of the
          proteins are merged together. Thus, we had to separate our target
          protein from the rest of the proteins that kept the cells functioning.
          To accomplish this, we had to depend upon the His-tags. Within the
          amino acids of our peptide sequence Histidine which contains polar
          properties. With this, we decided to use His-columns, columns that are
          lined on the edges with metal ions, to separate our target protein
          from the rest. The histidine within our protein sequence will be
          attracted to the metal ions on the sides of the column due to the
          bonds that will be formed between the Histidine and metal ions. Since
          the other protein doesn’t contain Histidine in its sequences, no
          bonding will occur which will cause it to continually float in the
          solution mixture, separating it from our target protein. As we wash
          away the solution mixture, none of our target protein will be removed
          since it&apos;s still bonded to the sides. Afterwards, we wash the
          column with imidazole, eluting the protein. The imidazole is capable
          of eluting the protein since it competes with the histidine to bind
          with the metal ions lining the his columns.
        </p>
        <div class="center-horizontal">
          <img
            src="https://static.igem.wiki/teams/4314/wiki/screen-shot-2565-10-12-at-18-41-24.png"
            class="fitimage1"
          />
          <p class="fitimage1">
            SDS-page after sonication of host cells. Targeted protein is shown
            as the highlighted bands. Other bands are different proteins that
            were present in host cells after Sonication.
          </p>
        </div>
        <div class="sizedbox20" />

        <p>
          <b>Results:</b>
        </p>
        <p class="indented">
          After eluting the column with imidazole, we ran the collected protein
          on gel electrophoresis. We determined if the protein is correct
          through the band sizes that are present. Our peptide, Bombolitin,
          requires band sizes of 125 bp.
        </p>
        <div class="center-horizontal">
          <img
            src="https://static.igem.wiki/teams/4314/wiki/results-contributions.png"
            class="fitimage1"
          />
          <p class="fitimage1">
            This shows that the protein that was collected was the correct one,
            with the base pairs of the bombolitin being between 100 and 200 base
            pairs.
          </p>
        </div>

        <p>
          <b>References:</b>
        </p>
        <p>
          &quot;His-Tag Purification.&quot; <i>Bio</i>,
          https://www.bio-rad.com/featured/en/his-tag-purification.html.
        </p>

        <div class="sizedbox40" />

        <h1 class="header1">Adding new parts to the iGEM registry</h1>
        <div class="sizedbox20"></div>
        <p class="indented">
          We contributed to future iGEM teams by submitting and characterizating
          bombolitin as a new parts:
          <a href="http://parts.igem.org/Part:BBa_K4314003" class="blue">
            <u>BBa_K4314003</u>
          </a>
          .
        </p>

        <div class="sizedbox40" />
        <h1 class="header1">
          Characterization and literature review on existing part BBa_J18918
        </h1>
        <div class="sizedbox20"></div>
        <p class="indented">
          We contributed to future iGEM teams by characterizing and performing a
          literature review on the part 
          <a href="http://parts.igem.org/Part:BBa_J18918" class="blue">
            <u>BBa_J18918</u>
          </a>
          .
        </p>

        <div class="sizedbox40" />
        <h1 class="header1">
          Characterization of BBa_J18918 by 2022 Thailand_RIS team
        </h1>
        <div class="sizedbox20"></div>
        <p>
          <b>Usage</b>
        </p>
        <p class="indented">
          For our project, we aimed to express an antimicrobial peptide (AMP), a
          modified version of bombolitin 
          <a href="http://parts.igem.org/Part:BBa_J18918" class="blue">
             
            <u>BBa_J18918</u>
          </a>
          , in <i>E. coli.</i> However, due to its antimicrobial properties, we
          could not express the modified bombolitin directly.
        </p>

        <div class="sizedbox20" />
        <p class="indented">
          The mechanism of action of the modified bombolitin lies in its
          positive charge and hydrophobicity, similar to how other AMPs act.
          Firstly, positively charged AMPs are attracted to the outer membrane
          protein of bacterial cells, since the majority of proteins embedded
          into the outer membrane of bacterial cells are negatively charged,
          thereby attracting the AMP’s negative charges. This causes the
          permeability of the membrane to be disrupted, effectively rupturing
          the membrane and killing the bacteria through lysis. Secondly,
          hydrophobic AMPs allow it to traverse the membrane and eventually get
          into the cytoplasm. Once in the cytoplasm, the AMP is able to disrupt
          a variety of intracellular functions, leading to eventual cell death
          [1]. Our modified bombolitin is both hydrophobic and cationic,
          allowing it to act intracellularly and on the membrane. Thus, it is
          extremely lethal to bacteria, including our expression host.
        </p>

        <div class="sizedbox20" />
        <p class="indented">
          To overcome this, fusion tags or stabilization tags can be added to
          the bombolitin in order to conceal its antimicrobial nature by
          reducing the hydrophobicity and the positive charge. For our project
          specifically, we used Thioredoxin. We also added a His-tag to allow us
          to separate the expressed bombolitin from other proteins of the
          expression host. Once expressed, we must separate the bombolitin
          sequence from the fusion tag. To do this, we added a TEV cleavage site
          between the His-tag and the bombolitin sequence, as shown below.
        </p>

        <div class="center-horizontal">
          <img
            src="https://static.igem.wiki/teams/4314/wiki/characterization-usage.png"
            class="fitimage1"
          />
          <p class="fitimage1">
            Using a TEV protease, we were able to separate the bombolitin
            sequence from the rest of the expressed sequence, allowing us to use
            E. coli to produce our modified bombolitin. Overall, we had very
            little trouble utilizing the TEV cleavage site and the TEV protease.
          </p>
        </div>

        <div class="sizedbox40" />
        <h1 class="header1">
          Literature Review of BBa_J18918 by 2022 Thailand_RIS team
        </h1>
        <div class="sizedbox20"></div>
        <p class="indented">
          Team Thailand_RIS (iGEM 2022) used the TEV cleavage site as a crucial
          part of our project, allowing us to express our modified bombolitin in 
          <i>E. coli</i>, despite its antimicrobial nature. Since TEV protease
          is widely used for cleaving fused proteins, we decided to do a
          literature review to add additional information to the registry page
          of this part. We hope that this will benefit future iGEM teams by
          allowing them to readily access important information regarding TEV
          protease and its cleavage site, as many future teams will likely use
          it as a vital part in their project.
        </p>

        <div class="sizedbox20" />
        <p>
          <b>Cleavage site:</b>
        </p>
        <p class="indented">
          The TEV protease recognizes the linear amino acid sequence
          E-X-X-Y-X-Q-(G/S/A). The cleaving is always performed between Q and
          G/S/A. The most efficient substrate for TEV protease is the cleavage
          site with amino acid sequence as follows: ENLYFQS. Substitutions in
          positions P2, P4, and P5 still allow cleavage by the TEV protease.
          However, even relatively conservative substitutions, such as Leu to
          Phe to P4, results in considerably lower efficiencies. On the other
          hand, G, S, and A can be interchanged without significant decrease in
          processing efficiency.
        </p>

        <div class="sizedbox20" />
        <p>Standard Conditions for Cleaving:</p>
        <p>
          Standard reaction buffer:
          <ul>
            <li>50 mM Tris-HCl (pH 8.0)</li>
            <li>0.5 mM EDTA</li>
            <li>1mM DTT</li>
          </ul>
        </p>

        <div class="sizedbox20" />
        <p>
          <b>Cleavage Inhibitors:</b>
        </p>
        <p class="indented">
          Because TEV protease is not part of the serine protease family, the
          following common protein inhibitors will not inhibit its cleaving:
          PMSF and AEBSF (1mM), TLCK (1mM), Bestatin (1mg/ml), pepstatin A
          (1mM), EDTA (1mM), and E-64 (3mg/ml). At concentrations of 5mM or
          above, Zinc ions will inhibit cleaving. Any reagents that react with
          cysteine will effectively inhibit the activity of TEV protease at the
          cleaving site. For instance, iodoacetamide or NEM.
        </p>
        <div class="sizedbox20" />
        <p>
          <b>References:</b>
        </p>
        <p>
          Fox, J. D. and Waugh, D. S. (2003). Maltose-binding protein as a
          solubility enhancer. <i>Methods Mol. Biol.</i> <b>205:</b> 99-117. 
          <br />
          <br />
          Kapust, R. B. and Waugh, D. S. (1999). Escherichia coli
          maltose-binding protein is uncommonly effective at promoting the
          solubility of polypeptides to which it is fused. <i>
            Protein Sci.
          </i> 
          <b>8:</b> 1668-1674. <br />
          <br />
          Kapust, R. B. and Waugh, D. S. (2000). Controlled intracellular
          processing of fusion proteins by <i>TEV protease.</i> Protein Expr.
          Purif. <b>19</b> : 312-318. <br />
          <br />
          Kapust, R. B., Tözsér, J., Fox, J. D., Anderson, D. E., Cherry, S.,
          Copeland, T. D., and Waugh, D. S. (2001). Tobacco etch virus protease:
          Mechanism of autolysis and rational design of stable mutants with
          wild-type catalytic proficiency. <i>Prot. Eng.</i> <b>14</b> :
          993-1000. <br />
          <br />
          Kapust, R. B., Tözsér, J., Copeland, T. D., and Waugh, D. S. (2002a).
          The P1&apos; specificity of tobacco etch virus protease. 
          <i>Biochem. Biophys. Res. Commun.</i> <b>294:</b> 949- 955. <br />
          <br />
          Kapust, R. B., Routzahn, K. M., and Waugh, D. S. (2002b). Processive
          degradation of nascent polypeptides, triggered by tandem AGA codons,
          limits the accumulation of recombinant TEV protease in Escherichia
          coli BL21(DE3).<i>Prot. Expr. Purif.</i> <b>24:</b> 61-70. <br />
          <br />
          Lucast, L. J., Batey, R. T., and Doudna, J. A. (2001). Large-scale
          purification of a stable form of recombinant tobacco etch virus
          protease. <i>Biotechniques</i> <b>30:</b> 544-550. <br />
          <br />
          Mohanty, A. K., Simmons, C.R., and Wiener, M.C. (2003). Inhibition of
          tobacco etch virus protease activity by detergents. 
          <i>Protein Expr. Purif.</i> <b>27:</b> 109-114. <br />
          <br />
          Phan, J., Zdanov, A., Evdokimov, A. G., Tropea, J. E., Peters, H. P.
          K., Kapust, R. B., Li, M., Wlodawer, A., and Waugh, D. S. (2002).
          Structural basis for the substrate specificity of tobacco etch virus
          protease. <i>J. Biol. Chem.</i> <b>277:</b> 50564-50572. <br />
          <br />
          Zhang, QY., Yan, ZB., Meng, YM. et al. (2021). Antimicrobial peptides:
          mechanism of action, activity and clinical potential. 
          <i>Military. Med. Res.</i> <b>8:</b> 48 <br />
          <br />
        </p>
      </div>

      <div class="sizedbox60" />
    </div>
  );
}

export default Contribution;
